deployment/terraform/cdn.tf

```diff
@@ -61,6 +61,28 @@ resource "aws_cloudfront_distribution" "cdn" {
     max_ttl                = 300
   }
 
+  ordered_cache_behavior {
+    path_pattern     = "tile/*"
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originAlb"
+
+    forwarded_values {
+      query_string = false
+      headers      = ["*"]
+
+      cookies {
+        forward = "all"
```

Comment:
It is possible that this setting should be changed to `none` or a specific set. When set to `all`, any values that differ between requests including the `Set-Cookie` header will get cached as separate objects.